app/views/users/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @user.name %>
+</p>
+
+<p>
+  <strong>Email:</strong>
+  <%= @user.email %>
+</p>
+
+  <h2 class="table-head padding">My created events</h2>
+  <% @user.own_events.each do |e| %>
+    <p><b>Event:</b> <%= e.title %> <b>Date:</b> <%= e.date %></p>
+  <% end %>
+
+  <h2 class="table-head padding">My Past Events</h2>
+
+  <% past = user_clasify_events(@user.attendances, 'past') %>
+  <% past.each do |e| %>
+    <p><b>Event:</b> <%= e.title %> <b>Date:</b> <%= e.date %></p>
+  <% end %>
+
+
+  <h2 class="table-head padding">My Upcoming Events</h2>
+  <% future = user_clasify_events(@user.attendances, 'future') %>
```

Comment:
**Affectations** is not allowed in views
Kindly use
```rb
```rb
current_user.events.upcoming_events
```